CHANGES.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## 1.11.0 (core) / 0.27.0 (libraries)
+
+## Major changes since 1.10.0 (core) / 0.26.0 (libraries)
+
+### Components — Configurable, reusable building blocks for data pipelines (now stable)
+
+Components, first released as Preview in 1.10.6, have reached Release Candidate status. APIs are stable, fully documented, and are the recommended, product-ready defaults for new Dagster projects.
+
+- **Declarative spec:** A short YAML block in `defs.yaml`, or a lightweight `Component` subclass in Python, lets you spin up assets, resources, schedules, IO managers, and checks, removing boilerplate while keeping every definition type-checked.
```

Comment:
Don't think "declarative spec" necessarily categorizes this bullet point the best. This is more "Simplified interface" or something like that.

Also might just be clearer to say "lets you spin up arbitrary dagster definitions, removing boilerplate while keeping all the flexibility of python"

Reply:
Also also, should decide if both words will be capitalized or just the first one in these headings, and be consistent throughout.

I don't particularly care either way, but I think we tend to just do the first one capitalized in our docs